specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -685,32 +789,37 @@ model TestRunArtifacts {
 @doc("The input artifacts for the test run.")
 model TestRunInputArtifacts {
```

Comment:
Q: should we we have different models for TestInputArtifacts and TestRunInputArtifacts?

Reply:
I dont find a reason for having separate models, but then we should rename the model to either InputArtifacts or TestAndTestRunInputArtifacts.